src/modules/cpu/kernel/image_sum.hpp

```diff
@@ -0,0 +1,846 @@
+#include "rppdefs.h"
+#include "rpp_cpu_simd.hpp"
+#include "rpp_cpu_common.hpp"
+#include "reduction.hpp"
+
+RppStatus image_sum_u8_u8_host_tensor(Rpp8u *srcPtr,
```

Comment:
this name is confusing for the api. please rename it to more meaningful name. Please don't combine image and tensor names in a single api unless it is converting from image to tensor or veceversa.
For exampe (tensor_sum_u32_u8_host). Also why are we outputting the sum into a float 32. It will loose precision

Reply:
Done - all names are now tensor_sum